docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -2747,7 +2747,6 @@ response.
 --------------------------------------------------
 GET /_nodes/stats?metric=ingest&filter_path=nodes.*.ingest
 --------------------------------------------------
```

Comment:
Just to say, I remove the AwaitsFix only from one of the node stats source code, so that we do not get a mass of test failures. With the next test failure, and after this PR is merged, the hope is that I can recognize where the "acknowledged: true" statement comes from since we will be printing also the YAML test source code file on failure.